CODEOWNERS

```diff
@@ -5,6 +5,6 @@
 # Order is important; for each modified file, the last matching
 # pattern takes the most precedence.
 
-/*                                         @johnlange2 @ryanhagen-tmo @eniolaodufuwa-tmo @jtbaumann @DRuffer-tmo @joebairdtmo @atrowbr2-tmo @wtucker6 @JLoucks4 @s-karstein @jvhutch @YHusain1
+/*                                         @johnlange2 @ryanhagen-tmo @eniolaodufuwa-tmo @jtbaumann @DRuffer-tmo @joebairdtmo @atrowbr2-tmo @wtucker6 @JLoucks4 @s-karstein @jvhutch @YHusain1 @rtalbott-tmo amovitz-tmobile rgrundy1
```

Comment:
Code owners contains errors

Reply:
add @ to the beginning of each username

Reply:
I fixed that, but I always see that error on this CODEOWNERS

Reply:
And I'm not sure "Undefined Kconfig symbols" can be fixed without catching up to the nightly merges 1st. Can we just merge this so I can move on and fix everything that way it should be?

Reply:
For some reason, I had to add the .github/workflows/tmo-nightly-merge-main-tmo-main.yml file back, but CODEOWNERS got resolved in the Catchup PR that was just merged. so This conversation is resolved.